multiraft/heartbeat_test.go

```diff
@@ -331,7 +328,6 @@ func TestHeartbeatMultipleGroupsJointLeader(t *testing.T) {
 			expCntSecondPhase, actCnt)
 	}
 	// Keep processing without inspection and shutdown cluster.
-	stopper.AddWorker()
 	go processEventsUntil(transport.Events, stopper, alwaysFalse)
```

Comment:
yeah, this goroutine can just die when the stopper has been invoked (because that means we're in the last line of the test anyways). In the previous code it was waiting to time out and only then let the stopper go ahead.
Could add a worker here again, but it shouldn't make a difference.